test/utils/format-unit-size.test.ts

```diff
@@ -0,0 +1,21 @@
+import { expect } from '@open-wc/testing';
+import { nothing } from 'lit';
+import { formatUnitSize } from '../../src/utils/format-unit-size';
+
+describe('formatUnitSize', () => {
+  it('returns blank when undefined number', () => {
+    expect(formatUnitSize(undefined, 1_0)).to.equal(nothing);
+  });
+
+  it('23 returns blank when undefined number', () => {
+    expect(formatUnitSize(1025, 1_0)).to.equal('1 kilobytes');
```

Comment:
If the numeric part is 1, the corresponding unit should not be plural. Can we get this to produce `1 kilobyte` without the `s`?

Also, I think the description for this test (23 / blank) doesn't describe what's actually happening.